book/_toc.yml

```diff
@@ -35,6 +35,7 @@
           sections:
             - file: tutorials/sar/sentinel1
             - file: tutorials/sar/uavsar
+            - file: tutorials/gpr/gpr
```

Comment:
note that this is currently under the SAR tutorial section. You'll want to move it up a level in the table of contents